<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewComposite</class>
 <widget class="QWidget" name="NewComposite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Композиты</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="2">
    <widget class="QComboBox" name="el1_cb">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="el1_pb">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="ionCharg_lb">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Заряд иона</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="create_pb">
     <property name="text">
      <string>Создать</string>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QLineEdit" name="el7_le"/>
   </item>
   <item row="17" column="0">
    <widget class="QPushButton" name="el9_pb">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="name_le">
     <property name="text">
      <string>MATERIAL</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="name_lb">
     <property name="text">
      <string>Наименование</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="el1_le"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="chemElem_lb">
     <property name="text">
      <string>Хим. элемент</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QLineEdit" name="el6_le"/>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="open_pb">
     <property name="text">
      <string>Открыть</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="el2_le"/>
   </item>
   <item row="13" column="2">
    <widget class="QComboBox" name="el6_cb">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QPushButton" name="el8_pb">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="16" column="2">
    <widget class="QComboBox" name="el8_cb">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="15" column="2">
    <widget class="QComboBox" name="el7_cb">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QPushButton" name="el7_pb">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="16" column="1">
    <widget class="QLineEdit" name="el8_le"/>
   </item>
   <item row="6" column="2">
    <widget class="QComboBox" name="el2_cb">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="mass_lb">
     <property name="text">
      <string>Массовая доля</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="el2_pb">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="19" column="2">
    <widget class="QComboBox" name="el11_cb">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="22" column="0">
    <widget class="QPushButton" name="el14_pb">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QPushButton" name="el5_pb">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QComboBox" name="el3_cb">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLineEdit" name="el3_le"/>
   </item>
   <item row="17" column="2">
    <widget class="QComboBox" name="el9_cb">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="el3_pb">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="21" column="2">
    <widget class="QComboBox" name="el13_cb">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="20" column="2">
    <widget class="QComboBox" name="el12_cb">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="23" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>66</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="17" column="1">
    <widget class="QLineEdit" name="el9_le"/>
   </item>
   <item row="21" column="1">
    <widget class="QLineEdit" name="el13_le"/>
   </item>
   <item row="20" column="0">
    <widget class="QPushButton" name="el12_pb">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="24" column="0" colspan="3">
    <widget class="QPushButton" name="save_pb">
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QPushButton" name="el4_pb">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="21" column="0">
    <widget class="QPushButton" name="el13_pb">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QComboBox" name="el4_cb">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="22" column="2">
    <widget class="QComboBox" name="el14_cb">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="20" column="1">
    <widget class="QLineEdit" name="el12_le"/>
   </item>
   <item row="18" column="2">
    <widget class="QComboBox" name="el10_cb">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <widget class="QComboBox" name="el5_cb">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="18" column="0">
    <widget class="QPushButton" name="el10_pb">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="22" column="1">
    <widget class="QLineEdit" name="el14_le"/>
   </item>
   <item row="13" column="0">
    <widget class="QPushButton" name="el6_pb">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="19" column="0">
    <widget class="QPushButton" name="el11_pb">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="18" column="1">
    <widget class="QLineEdit" name="el10_le"/>
   </item>
   <item row="23" column="0">
    <widget class="QPushButton" name="check_pb">
     <property name="text">
      <string>Проверить</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLineEdit" name="el5_le"/>
   </item>
   <item row="19" column="1">
    <widget class="QLineEdit" name="el11_le"/>
   </item>
   <item row="23" column="1">
    <widget class="QPushButton" name="ioniz_pb">
     <property name="text">
      <string>Ионизированный</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLineEdit" name="el4_le"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="density_lb">
     <property name="text">
      <string>Плотность</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="density_le"/>
   </item>
   <item row="0" column="2" rowspan="4">
    <spacer name="horizontalSpacer">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>66</width>
       <height>78</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
